storage/replica_command.go

```diff
@@ -743,6 +743,19 @@ func (r *Replica) PushTxn(batch engine.Engine, ms *engine.MVCCStats, args proto.
 	if err != nil {
 		return reply, err
 	}
+	// There are three cases in which there is no transaction entry, in decreasing likelihood
+	// * a transaction is still active; it just hasn't committed/restarted/aborted/heartbeat yet
+	// * the transaction resolved its intents synchronously on successful commit; in this case,
+	// the transaction record is also removed. Note that in this case, the intent which prompted
```

Comment:
check that for multi-line bullets, the extra lines are properly indented to line up with the first word on the first line.